src/components/Capture/capture.js

```diff
@@ -60,23 +61,42 @@ class Capture extends Component {
     checkIfHasWebcam( hasWebcam => this.setState({hasWebcam}) )
   }
 
-  validateCapture = (id, valid) => {
-    const { actions, method } = this.props
-    actions.validateCapture({ id, valid, method})
+  maxAutomaticCaptures = 3
+
+  uploadCaptureToOnfido = () => {
+    this.setState({uploadInProgress: true})
+    const {validCaptures, method, side, token, nextStep,
+      actions: {confirmCapture}
+    } = this.props
+    const capture = validCaptures[0]
+    postToOnfido(capture, method, token, this.state.advancedValidation,
+      (apiResponse) => {
```

Comment:
I find it hard to read when a function is defined in the middle of a list of arguments - can we assign it to a variable and pass it in instead?